tests/_plugins/stateless/test_mpl.py

```diff
@@ -64,3 +64,19 @@ async def test_mpl_show(k: Kernel, exec_req: ExecReqProvider) -> None:
             )
         ]
     )
+
+
+@pytest.mark.skipif(
+    not DependencyManager.matplotlib.has(),
+    reason="matplotlib is not installed",
+)
+def test_patch_javascript() -> None:
+    from matplotlib.backends.backend_webagg_core import FigureManagerWebAgg
+
+    from marimo._plugins.stateless.mpl._mpl import patch_javascript
+
+    javascript: str = str(FigureManagerWebAgg.get_javascript())  # type: ignore[no-untyped-call]
+    assert javascript is not None
+    javascript = patch_javascript(javascript)
+    assert javascript.count("// canvas.focus();") == 1
+    assert javascript.count("// canvas_div.focus();") == 1
```

Comment:
I _think_ this is farily robust to ensure the script doesn't change out from underneath us. Would you want a find+replace with more context to ensure we make the change in one spot?

Reply:
Sure, I also don't know if more or less fragile so sticking with it makes sense.